app/schema.graphql

```diff
@@ -1086,6 +1086,11 @@ type JSONInvocationParameter implements InvocationParameterBase {
   defaultValue: JSON
 }
 
+type JSONPromptMessageGQL {
```

Comment:
Can we remove `GQL` from the type names in the GraphQL schema?

Reply:
Agreed